mobile/components/AudioPlayer.tsx

```diff
@@ -0,0 +1,55 @@
+import React, { useState, useEffect } from "react";
+import { View } from "tamagui";
+import { Audio } from "expo-av";
+import { Icon } from "./Icon";
+
+interface AudioPlayerProps {
+  uri: string;
+}
+
+/**
+ * TODO: This component would need support for integrating plyaback controls.
+ * For now, it just plays the audio file.
+ * Recommended: react-native track player
+ */
+const AudioPlayer = (props: AudioPlayerProps) => {
+  const { uri } = props;
+  const [sound, setSound] = useState<Audio.Sound | undefined>(undefined);
+  const [status, setStatus] = useState<any>({});
```

Comment:
we should avoid using any. please update with an actual type.

Reply:
👍 done